types/src/form.rs

```diff
@@ -130,6 +130,11 @@ impl AnnotatedForm {
     async fn document_id(&self) -> DocumentId {
         self.position.document_id
     }
+
+    /// Unique identifier of this form
+    async fn id(&self) -> Uuid {
+        self.id.unwrap_or_default()
```

Comment:
Definitely don't use `unwrap_or_default()` here because that'll return a zeroe'd out UUID that doesn't mean anything if `self.id` is `None`. Words coming from the DB should _always_ have `Some(id)`, only ones being inserted have `None`. So you can throw an error here by making the return value `FieldResult<Uuid>` and using `anyhow` to produce an error if it's `None`.